NSrtm.Core/Pgm/GridGraph/IPgmGridGraph.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace NSrtm.Core.Pgm.GridGraph
+{
+    public interface IPgmGridGraph
+    {
+        double GetUndulation(double latitude, double longitude);
```

Comment:
does not require. Currently a value from the nearest corner will be selected. In the future, interpolation will be used to approximate the geoid undulation.